specification/indices/simulate_index_template/IndicesSimulateIndexTemplateRequest.ts

```diff
@@ -64,4 +65,6 @@ export interface Request extends RequestBase {
      */
     include_defaults?: boolean
   }
+  /** @codegen_name index_template */
+  body?: IndexTemplate
```

Comment:
https://github.com/elastic/elasticsearch/blob/63840f1276b1fea156a96a6ae16380c381982900/server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestSimulateIndexTemplateAction.java#L50-L61

This fixes a YAML test: https://github.com/elastic/elasticsearch/blob/63840f1276b1fea156a96a6ae16380c381982900/modules/data-streams/src/yamlRestTest/resources/rest-api-spec/test/data_stream/230_data_stream_options.yml#L173-L176